recipes/_start.rb

```diff
@@ -10,6 +10,16 @@
   not_if { platform_family?('windows') }
 end
 
+ruby_block 'insert ulimit' do
+  block do
+    file = Chef::Util::FileEdit.new('/etc/init.d/splunk')
+    file.insert_line_after_match(/^RETVAL=\d$/, "ulimit -n #{node['splunk']['limits']['open_files']}")
```

Comment:
What are the odds that splunk would every modify the first line of the init.d file to something other than `RETVAL=\d`?  I would be more comfortable with this if we had a serverspec to cover it in an integration test.  Otherwise we may be better off doing something like [this](http://stackoverflow.com/questions/8623231/prepend-a-single-line-to-file-with-ruby) to simply prepend that line to the file.